src/libs/actions/Report.js

```diff
@@ -249,6 +249,7 @@ function addActions(reportID, text = '', file) {
 
     const optimisticReport = {
         lastVisibleActionCreated: currentTime,
+        lastMessageTranslationKey: lodashGet(lastAction, 'message[0].translationKey', ''),
```

Comment:
It looks like that push event should also update this field to update let LHN message after sending an attachment, like this case https://github.com/Expensify/App/issues/22937